src/components/DroughtHistory.vue

```diff
@@ -177,7 +180,7 @@ export default {
           .attr("width", '100%')
 
         // Define y scale based on timeline start and end dates
-        const timelineDates = ['1950-04-01','2020-03-22']
+        const timelineDates = ['1920-10-01','2021-03-31']
```

Comment:
Ok, so this is a bit weird and I'm not sure what to do about it. I have to have inconsistent timeline here in `vue` compared to the actual plotting function because it makes everything line up. The upside down years are from turning on the labels on the R-based `png` and the right-side up years are from the `vue` code. You can see here, they line up:

<img width="1428" alt="image" src="https://github.com/USGS-VIZLAB/drought-timeline/assets/19958841/d00d13cf-6e9b-4f58-9b1d-d7291e292f32">


Now, when I make the `vue` timeline consistent with the scales on the `png`:

<img width="1428" alt="image" src="https://github.com/USGS-VIZLAB/drought-timeline/assets/19958841/709a2db7-2c63-42bf-b679-36a6af95c330">

Reply:
Got it sorted! PR [here](https://github.com/aaarcher-usgs/drought-timeline/pull/1)